### DISCUSSION TITLE
Kiosk URL updated

### DISCUSSION THREAD

**Discussion by MaBeniu** [tijsverkoyen/HomeAssistant-FusionSolar#156] [Q&A] [answered]
Hi, I am using integration with Kiosk option. How to update integration setup to not loose data after kiosk URL is changed instead of removing old and adding new. Please help.
I changed core.config_entries. restarted HA and it automatically created new entities :(

**Comment by tijsverkoyen** [comment]
There is no easy way to do this. You can google to see how to migrate data from old to new entities. But at your own risk. So make sure you have backups.

**Reply by StefanLobbenmeier** [reply]
Is updating the kiosk url something that is just not implemented right now or is this something that can never be supported because of home assistant limitations?

**Reply by tijsverkoyen** [reply]
~Not really, because I use the the id in the kiosk url to generate unique entity names.~

**Reply by tijsverkoyen** [reply, marked as answer]
See https://github.com/tijsverkoyen/HomeAssistant-FusionSolar/?tab=readme-ov-file#i-need-to-change-the-kiosk-url
